ui/components/institutional/jwt-dropdown/jwt-dropdown.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Dropdown from '../../ui/dropdown';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+const JwtDropdown = (props) => {
+  const t = useI18nContext();
+  const { currentJwt, jwtList } = props;
+
+  return (
+    <div>
+      <span className="custody-search-jwt__select-title">{t('selectJWT')}</span>
```

Comment:
For this we should use the new `<Text>` design system component with `padding={3}` and `color={Color.___}` values.